docs/api/context.md

```diff
@@ -12,7 +12,7 @@
   snippet:
 
 ```js
-app.use(function *(){
+app.use(function * () {
```

Comment:
In v2 we only use it [once](https://github.com/koajs/koa/blob/v2.x/test/application/use.js#L66) without space. Usage in docs is [inconsistent](https://github.com/koajs/koa/pull/686/files/340dd4f1a3c0b715e259bbcbea9d05aeb79619c1#diff-0f569af1b88af54931448ef15c1c26b1R151).

In v1 it is sometimes [with space](https://github.com/koajs/koa/blob/master/test/request/href.js#L29) but [mostly](https://github.com/koajs/koa/blob/master/test/application.js#L17) without.

In standard.js it is [defined](https://github.com/feross/eslint-config-standard/blob/master/eslintrc.json#L41) to have [spaces around tha star](http://eslint.org/docs/rules/generator-star-spacing) and that is what I used.

I can unify it if it is clear what to use.